reference/api/index.md

```diff
@@ -3,3 +3,20 @@ title: API reference
 layout: default
 ---
 This is a reference for the Zetkin Platform API.
+
+
+## General information
+
+###  Pagination
+
+Collections can be paginated by supplying paging parameters to the request.
+
+- p: Page. The requested page index.
+- pp: Per page. Number of items per page.
+
+```json
```

Comment:
The parameters should be supplied as JSON, but as query parameters. A better example would be to use an existing collection (while linking to it) and describe it's pagination here. Something like this:

> The parameters should be supplied in the query string. For example, if you want to paginate the [people collection](people), make a request to `/orgs/1/people?p=2&pp=20`.